README.md

```diff
@@ -0,0 +1,49 @@
+# Google Code-in Leaders
+***
+This is a website to show the current leaders from all organizations in GCI.
+***
+ * [Installation](#installation)
+ * [Languages](#languages)
+ * [Usage](#usage)
+ * [Getting Involved](#getting-involved)
+ * [Support](#support)
+ * [Authors](#authors)
+ * [License](#license)
+***
+### Installation
+```
+npm install
+```
+***
+### Languages:
+* Node.js
+* JSON
+***
+### Usage:
+```
+npm run build
+```
+***
+### Getting Involved
+If you would like to be a part of the coala community, you can check out our [Getting In Touch](http://coala.readthedocs.io/en/latest/Help/Getting_In_Touch.html) page or ask us at our active Gitter channel, where we have maintainers from all over the world. We appreciate any help!
+
+We also have a [newcomer guide](http://api.coala.io/en/latest/Developers/Newcomers_Guide.html) to help you get started by fixing an issue yourself! If you get stuck anywhere or need some help, feel free to contact us on Gitter or drop a mail at our [newcomer mailing list](https://groups.google.com/forum/#!forum/coala-newcomers).
```

Comment:
as @yukiisbored mentioned we don't use mailing list, you should remove it from here too.